src/net.cpp

```diff
@@ -1510,7 +1510,6 @@ static std::string GetDNSHost(const CDNSSeedData &data, uint64_t requiredService
     // use default host for non-filter-capable seeds or if we use the default service bits (NODE_NETWORK)
     if (!data.supportsServiceBitsFiltering || requiredServiceBits == NODE_NETWORK)
     {
-        requiredServiceBits = NODE_NETWORK;
```

Comment:
correct. cause `requiredServiceBits` to NODE_NETWORK just before calling `GetDNSHost()` at line 1571